cmd/sifgen/main.go

```diff
@@ -12,53 +11,27 @@ import (
 func main() {
 	rootCmd := &cobra.Command{Use: "sifgen"}
 
-	_networkCmd := networkCmd()
-	_networkCmd.AddCommand(networkCreateCmd(), networkResetCmd())
-
 	_nodeCmd := nodeCmd()
 	_nodeCreateCmd := nodeCreateCmd()
+	_nodeCreateCmd.PersistentFlags().Bool("standalone", false, "standalone node")
+	_nodeCreateCmd.PersistentFlags().String("admin-clp-addresses", "", "admin clp addresses")
+	_nodeCreateCmd.PersistentFlags().String("admin-oracle-address", "", "admin oracle addresses")
+	_nodeCreateCmd.PersistentFlags().String("bind-ip-address", "127.0.0.1", "IPv4 address to bind the node to")
+	_nodeCreateCmd.PersistentFlags().String("peer-address", "", "peer node to connect to")
+	_nodeCreateCmd.PersistentFlags().String("genesis-url", "", "genesis URL")
+	_nodeCreateCmd.PersistentFlags().String("bond-amount", "100000000000000000rowan", "bond amount")
```

Comment:
Hey is this the min bond-amount?

Reply:
No, it's just an initial amount for the standalone validator (when we're initialising a new network) to use to bond/stake.